test/JDBC/expected/four-part-names-vu-verify.out

```diff
@@ -0,0 +1,373 @@
+CREATE TABLE fpn_table (a int, b varchar(10))
```

Comment:
No, this won't entirely work because if there are objects dependent (in this test file, it is a view) on OPENQUERY() (or four-part names from now on), it will fail MVU. Apart from this there is no real upgrade scenario to be tested and there are no upgrade script changes as well; that's why I did not add the upgrade tests.

Reply:
We should not assume any MVU failures in Babelfish. Have you discussed this behavior with @surendravishnoi and @rishabhtanwar29?